modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/WorkflowUtil.java

```diff
@@ -34,10 +34,9 @@ private WorkflowUtil() {
    *          The workflow state to check.
    * @return True if the workflow is currently active, not stopped or failed.
    */
+  @Deprecated
   public static boolean isActive(WorkflowState workflowState) {
-    return WorkflowState.INSTANTIATED.equals(workflowState)
-            || WorkflowState.RUNNING.equals(workflowState)
-            || WorkflowState.PAUSED.equals(workflowState);
+    return !workflowState.isTerminated();
   }
 
   /**
```

Comment:
The `#isActive(String)` method needs to be updated as well.

Reply:
done.